src/discord-cluster-manager/run_hello_world_workflow.py

```diff
@@ -0,0 +1,37 @@
+from consts import GITHUB_REPO, GITHUB_TOKEN
```

Comment:
should this go in `scripts/`?

Reply:
Ideally, it should go in `scripts/`. The awkward thing about `scripts/` is that there's no easy way to import these constants (or am I missing something?). On the other hand, I don't really need the constants (I can use `os.getenv(...)` instead), I can use `print` instead of `setup_logging`, and I don't need `get_github_branch_name`. Let me try moving it to scripts and see if that's not too bad.

Reply:
The new commit moves it to `scripts/` - @msaroufim, please have another look when you have a moment.